pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUmbrellaStudyI18n.java

```diff
@@ -23,6 +24,17 @@
     @RegisterRowMapper(UmbrellaStudyI18nDtoMapper.class)
     List<StudyI18nDto> findTranslationsByStudyId(@Bind("id") long id);
 
+    @SqlQuery("SELECT lc.iso_language_code, i18n.name, i18n.summary "
+            + "FROM i18n_umbrella_study AS i18n JOIN language_code AS lc "
+            + " ON i18n.language_code_id = lc.language_code_id"
+            + " WHERE i18n.umbrella_study_id = :id AND lc.language_code_id = :langCodeId"
+    )
+    @RegisterRowMapper(UmbrellaStudyI18nDtoMapper.class)
+    Optional<StudyI18nDto> findTranslationByStudyIdAndLanguageCodeId(
+            @Bind("id") long id,
+            @Bind("langCodeId") long languageCodeId
+    );
```

Comment:
This method is needed to fetch the translation for the language taken from the attribute store.